conf/docker/dataverse-glassfish/init-container/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM centos:7.2.1511
```

Comment:
Why this level of specification for the OS version?

Reply:
I'm sure this is a copy paste from either of

- conf/docker/dataverse-glassfish/Dockerfile
- conf/docker/solr/Dockerfile

@pameyer does it make sense to bump all of these to a newer version? Does Docker just let you say 7.x? Is that a bad idea? Should it be fixed? I'm not sure how security patches are applied to Docker containers when CentOS releases them.

Reply:
`centos:7` or `centos:7.1` should both be accepted.  My understanding is that patches get applied with the image is rebuild if there's a patched copy of the base image available, but that could be inaccurate.

Reply:
@thaorell  This should be centos:7